# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1356124293 18000
#      Fri Dec 21 16:11:33 2012 -0500
# Node ID f366e9948259f48c9743ee010c0c0b89c5a619db
# Parent  abc9e5f3c6613fbd1017fb131f463efa01e39cd8
# Parent  23a180fd186d8e7f80c248119ac6daee66aa950c
periodic merge of stable to default

diff --git a/doc/interpreter/Makefile.am b/doc/interpreter/Makefile.am
--- a/doc/interpreter/Makefile.am
+++ b/doc/interpreter/Makefile.am
@@ -284,12 +284,12 @@ EXTRA_DIST = \
   $(IMAGES) \
   $(IMAGES_SRC) \
   $(LOGOS) \
   $(TXI_SRC)
 
 clean-local:
 	rm -rf t2d_cache
 
-DISTCLEANFILES = $(BUILT_TEXINFOS) doc-cache
+DISTCLEANFILES = $(BUILT_TEXINFOS)
 
-MAINTAINERCLEANFILES = $(BUILT_IMAGES)
+MAINTAINERCLEANFILES = $(BUILT_IMAGES) doc-cache
 
